R/available.R

```diff
@@ -114,3 +109,39 @@ suggest <- function(path = ".",  field = c("Title", "Description"), text = NULL)
 
   namr(text)
 }
+
+
+
+check_online_terms <- function(term, urban = TRUE) {
+  if (urban) {
```

Comment:
How about this instead, if urban is FALSE the expression will return NULL, which will be ignored by the `compact()`.

```r
compact(list(
  get_bad_words(term),
  get_abbreviation(term),
  get_wikipidia(term),
  get_wiktionary(term),
  if (urban) get_urban_data(term),
  get_sentiment(term))
)
```

Reply:
Oh, I didn't know it